wsgi/odyssy/basic/templates/basic/index.html

```diff
@@ -36,17 +36,17 @@
             <!-- Announcements -->
             <div class="col-sm-4" id="announcements">
                 <div class='announcement-content '>
-                    <div class="widgets-title"><h2>Announcements</h2></div>
+                    <div class="widgets-title"><h2><a href="/announcement/">Announcements</a></h2></div>
```

Comment:
@Monal5031 I wouldn't, but there's no view created for all announcements, Is there?